setup.py

```diff
@@ -36,7 +36,7 @@ def get_long_description():
         'thoth.package_extract.handlers',
     ],
     package_data={
-        'thoth-package-extract': os.path.join('thoth', 'package_extract', 'bin', 'skopeo')
+        'thoth.package_extract': [os.path.join('bin', 'skopeo')]
```

Comment:
Shouldn't the path be relative to the `setup.py` file? I.e `os.path.join('thoth', 'package_extract', 'bin', 'skopeo')`?

Reply:
In [docs](https://docs.python.org/3.6/distutils/setupscript.html#installing-package-data): 

> The value must be a mapping from package name to a list of relative path names that should be copied into the package. The paths are interpreted as relative to the directory containing the package (information from the package_dir mapping is used if appropriate); that is, the files are expected to be part of the package in the source directories.

Reply:
Got it. :+1: